From d5a81b26b84965045cab6b74a928bb78f019dcda Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Mon, 30 Jul 2012 08:44:08 -0700
Subject: [PATCH 113/795] MR1: MAPREDUCE-4417. add support for encrypted shuffle (tucu)

  Reason: Amendment to testcase to initialize MapOutputServlet outside of the TastTracker
  Author: Alejandro Abdelnur
  Ref: CDH-7046
(cherry picked from commit b8545d306c2c08febbae199c82fad61607bc8c75)

(cherry picked from commit 631932194ec6bec5974a365ef14f69b430872bcd)
---
 .../apache/hadoop/mapred/TestShuffleJobToken.java  |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/test/org/apache/hadoop/mapred/TestShuffleJobToken.java b/src/test/org/apache/hadoop/mapred/TestShuffleJobToken.java
index 70722dc..d4c69b3 100644
--- a/src/test/org/apache/hadoop/mapred/TestShuffleJobToken.java
+++ b/src/test/org/apache/hadoop/mapred/TestShuffleJobToken.java
@@ -68,6 +68,7 @@ public void setUp() throws Exception {
     }
     server = new HttpServer("test", "0.0.0.0", 0, true);
     server.addServlet("shuffle", "/mapOutput", TaskTracker.MapOutputServlet.class);
+    server.setAttribute(JobTracker.SHUFFLE_SSL_ENABLED_KEY, false);
     server.start();
     int port = server.getPort();
     baseUrl = new URL("http://localhost:" + port + "/");
-- 
1.7.0.4

